src/test.js

```diff
@@ -4,6 +4,8 @@ console.log('Running test');
 
 console.log('cpuRdtsc', vm_detection.cpuRdtsc(false, 10, 500));
 console.log('cpuRdtsc+vm exit', vm_detection.cpuRdtsc(true, 10, 500));
+console.log('cpuRdtsc+max interval', vm_detection.cpuRdtsc(true, 10, 5000));
+console.log('cpuRdtsc+max samples+max interval', vm_detection.cpuRdtsc(true, 10 * 1024 * 1024, 5000));
```

Comment:
yes, this conforms perfectly to the level of testing that puts this library in such high regard amongst its users

Reply:
I live to serve